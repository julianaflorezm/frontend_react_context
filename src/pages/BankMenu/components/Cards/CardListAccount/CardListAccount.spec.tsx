import { render } from '@testing-library/react';
import { AccountCardListProps, CardListAccount } from './index';

describe('CardListAccount Test', () => {
  let props: AccountCardListProps;

  beforeEach(() => {
    props = {
      selected: 1,
      accounts: [
        {
          id: 1,
          nombre: 'Cuenta de ahorro',
          numeroCuenta: '234vf5',
          saldo: 50000,
          fechaCreacion: '2022-03-05 11:00:00',
        },
        {
          id: 2,
          nombre: 'Cuenta de ahorro',
          numeroCuenta: 'fgt65yg',
          saldo: 50000,
          fechaCreacion: '2022-03-04 11:00:00',
        },
      ],
      setSelected: jest.fn(),
    };
  });

  it('should match snapshot', () => {
    const { container } = render(<CardListAccount {...props} />);
    expect(container).toMatchSnapshot();
  });
});
