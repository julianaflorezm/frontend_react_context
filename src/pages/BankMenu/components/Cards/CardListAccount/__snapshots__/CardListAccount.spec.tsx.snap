// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`CardListAccount Test should match snapshot 1`] = `
<div>
  <div
    class="MuiPaper-root MuiCard-root makeStyles-containerSelected-2 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <button
      class="MuiButtonBase-root MuiCardActionArea-root"
      tabindex="0"
      title="action"
      type="button"
    >
      <div
        class="MuiCardContent-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
        >
          <span
            title="nombre"
          >
            Cuenta de ahorro
             con referencia:
          </span>
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            title="numero"
          >
            234vf5
          </span>
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2"
        >
          <span
            title="saldo"
          >
             Saldo disponible: $
            50000
          </span>
           
        </p>
      </div>
      <div
        class="MuiCardActions-root makeStyles-position-3 MuiCardActions-spacing"
      >
        <a
          class="makeStyles-linkSelected-5"
          href="#"
        >
          Ver movimientos
        </a>
      </div>
      <span
        class="MuiCardActionArea-focusHighlight"
      />
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
  <div
    class="MuiPaper-root MuiCard-root makeStyles-container-1 MuiPaper-elevation1 MuiPaper-rounded"
  >
    <button
      class="MuiButtonBase-root MuiCardActionArea-root"
      tabindex="0"
      title="action"
      type="button"
    >
      <div
        class="MuiCardContent-root"
      >
        <p
          class="MuiTypography-root MuiTypography-body1 MuiTypography-gutterBottom"
        >
          <span
            title="nombre"
          >
            Cuenta de ahorro
             con referencia:
          </span>
        </p>
        <p
          class="MuiTypography-root MuiTypography-body1"
        >
          <span
            title="numero"
          >
            fgt65yg
          </span>
        </p>
        <p
          class="MuiTypography-root MuiTypography-body2"
        >
          <span
            title="saldo"
          >
             Saldo disponible: $
            50000
          </span>
           
        </p>
      </div>
      <div
        class="MuiCardActions-root makeStyles-position-3 MuiCardActions-spacing"
      >
        <a
          class="makeStyles-link-4"
          href="#"
        >
          Ver movimientos
        </a>
      </div>
      <span
        class="MuiCardActionArea-focusHighlight"
      />
      <span
        class="MuiTouchRipple-root"
      />
    </button>
  </div>
</div>
`;
